secret.config.js

```diff
@@ -0,0 +1 @@
+process.env.MINTBASE_API_KEY=e886efa80e627f165ebbc7b4d942c172
```

Comment:
Should this be public?

Reply:
This key itself isn't sensitive but may be in the future, seems like this PR might now be missing commits :/

Reply:
Ah, nm we are just using .env via dotconfig now